fpdf/fpdf.py

```diff
@@ -1981,6 +1982,12 @@ def set_stretching(self, stretching):
         if self.page > 0:
             self._out(f"BT {stretching:.2f} Tz ET")
 
+    def set_fallback_fonts(self, fallback_fonts):
```

Comment:
maybe `add_fallback_fonts()` ?
That would give the users more flexibility, in case they discover they need to add another font later on.

Reply:
`add_fallback_fonts()`, as I understand it, could be a bit limiting, as it would not allow to remove fallback fonts, for example in some sections of a document.
In that sense `set_fallback_fonts()` would provide a bit more control to the users: they could call it with more or less fonts, and even with `[]` to remove all fallback fonts.